c7n/filters/core.py

```diff
@@ -103,6 +113,8 @@ def intersect(x, y):
     'in': operator_in,
     'ni': operator_ni,
     'not-in': operator_ni,
+    'not-in-list-item': operator_ni_list_elem,
```

Comment:
maybe instead call them `in-each` and `not-in-each`?

Reply:
ahh much better. had trouble thinking of a good name